src/ApiMethodDocumentation.js

```diff
@@ -397,6 +397,8 @@ export class ApiMethodDocumentation extends AmfHelperMixin(LitElement) {
       return;
     }
     this._server = value;
+    this.baseUri = this._getBaseUri(undefined, value);
```

Comment:
this cannot be done like this because you are overriding the `baseUri` property. At this point you can't be sure whether the `baseUri` came from server definition or from actually setting a `baseUri`. 
Base URI has a priority over the server definition and should be passed to the view if set. When no `baseUri` exists then it should be computed from the `server`, if available. 
I would add a read only private property like _serverBaseUri that holds a computed value of server defined base URI (should be cleared when server has no value). Then a getter that returns the effective base URI which is either (in order): set attribute base URI, server base URI, or undefined. This effective base URI then should be used in `_computeEndpointUri` function to compute the URI value.